extruct/jsonld.py

```diff
@@ -15,6 +15,12 @@
 class JsonLdExtractor(object):
     _xp_jsonld = lxml.etree.XPath('descendant-or-self::script[@type="application/ld+json"]')
 
+    def __init__(self, allow_control_characters=True):
+        if allow_control_characters:
```

Comment:
You could even set `self.strict = not allow_control_characters` to simplify this.